tests/libxml2/conf.yml

```diff
@@ -3,6 +3,16 @@ transpile:
   binary: runtest
   tflags: --reorganize-definitions
 
+refactor:
+  autogen: true
+  transforms:
+  - remove_unused_labels
+  - remove_null_terminator
```

Comment:
I missed this during review so it's on me, but: these transforms don't do anything because they require the `target` mark and we don't provide that. I opened https://github.com/immunant/c2rust/issues/1493 to remove them.